### ISSUE TITLE
Edit Node on request is not populating request description.

### ISSUE THREAD

**Issue by AnujaK** [AnujaK/restfiddle#243]
Even though request has a description, it doesn't appear in the modal. If new description is entered in modal, it replaces the previous one.
Assigned to @kshijiD

**Comment by kshijiD** [comment]
Hi,

  I have debugged the entire flow. For the above bug there some issue at
back end side. On backend side in NodeController In method wherein  we
fetch the tree node and return them I tried printing the rootNode the desc
is null for all nodes. So on UI Side the data I'am getting  from server
side is incorrect.


On Mon, Apr 27, 2015 at 3:53 PM, Anuja Kumar <notifications@github.com>
wrote:

> Even though request has a description, it doesn't appear in the modal. If
> new description is entered in modal, it replaces the previous one.
> Assigned to @kshijiD <https://github.com/kshijiD>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AnujaK/restfiddle/issues/243>.
>



-- 





 kshitija dharwadkar   |   web engineer   |   raw engineering
kshitija.dharwadkar@raweng.com <carla@raweng.com>  |   skype: kshitija912
